src/main.c

```diff
@@ -213,17 +213,26 @@ void c_main(void)
 	 * miss and therefore cause no evictions.
 	 */
 
+	int id = MMC_ID;
 	mmc_inited = !mmc_init(MMC_ID);
+	char *rootfs_dev = "root=/dev/mmcblk0p1";
+#ifdef MMC_SD
+	if (!mmc_inited) {
+		rootfs_dev = "root=/dev/mmcblk1p1";
+		id = MMC_ID_SD;
+		mmc_inited = !mmc_init(MMC_ID_SD);
+	}
+#endif
```

Comment:
You might also want to add a "alt4_key_pressed" function in board-lepus.c, so that when a specific button is pressed (e.g. B), it will boot to the external SD card by default.